pkg/aws/sts/arn_resolver.go

```diff
@@ -30,6 +30,11 @@ func DefaultResolver(prefix string) *Resolver {
 
 // Resolve converts from a role string into the absolute role arn.
 func (r *Resolver) Resolve(role string) string {
+
```

Comment:
@Joseph-Irving remove this whitespace line?

Reply:
yep done